mne_bids_pipeline/steps/preprocessing/_03_maxfilter.py

```diff
@@ -340,9 +340,21 @@ def run_maxwell_filter(
         verbose=cfg.read_raw_bids_verbose,
     )
     bids_path_ref_bads_in = in_files.pop("raw_ref_run-bads", None)
+    # load head pos
+    if cfg.mf_mc:
+        head_pos = mne.chpi.read_head_pos(in_files.pop(f"{in_key}-pos"))
+    else:
+        head_pos = None
+    # triage string-valued destinations
     if isinstance(destination, str):
-        assert destination == "reference_run"
-        destination = raw.info["dev_head_t"]
+        if destination == "reference_run":
+            destination = raw.info["dev_head_t"]
+        elif destination == "twa_hp":
+            assert head_pos is not None, (
+                "cannot compute time-weighted average head position without movement "
+                "compensation. Please set `mf_mc=True` in your config file."
+            )
```

Comment:
I think ideally this would happen in the config validation code I think rather than here, that way a user gets immediate feedback when starting their pipeline run rather than (perhaps much) later after some steps are potentially performed.